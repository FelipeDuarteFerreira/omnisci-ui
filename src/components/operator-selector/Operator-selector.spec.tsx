import React from "react"
import { render } from "enzyme"

import OperatorSelector from "./Operator-selector"

test("Should render an operator selector", () => {
  const props = {
    menuIsOpen: false,
    operator: "between",
    min: 0,
    max: 10
  }
  const component = render(<OperatorSelector {...props} />)

  expect(component).toMatchSnapshot()
})
