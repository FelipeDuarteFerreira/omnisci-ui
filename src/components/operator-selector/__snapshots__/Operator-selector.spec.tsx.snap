// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render an operator selector 1`] = `
<div
  class="operator-selector"
>
  <div
    class="operator"
  >
    <div
      class="label"
    >
      between
    </div>
    <div
      class="arrow"
    />
  </div>
  <div
    class="min-input"
  >
    <input
      type="text"
      value="0"
    />
  </div>
  <div
    class="between-label"
  >
    and
  </div>
  <div
    class="max-input"
  >
    <input
      type="text"
      value="10"
    />
  </div>
</div>
`;
